packages/core/lib/database/updateErrorListRecord.ts

```diff
@@ -16,7 +18,6 @@ const generateUpdateFields = (result: PhaseResult): Partial<ErrorListRecord> =>
     error_quality_checked: record.error_quality_checked,
     annotated_msg: record.annotated_msg,
     updated_msg: record.updated_msg,
-    error_status: ResolutionStatus.UNRESOLVED,
```

Comment:
Where does the error_status get set now?

Reply:
```
      UPDATE br7own.error_list SET ${db(updateFields)},
        error_status = CASE
          WHEN ${exceptionsCount}::integer > 0 THEN ${ResolutionStatus.UNRESOLVED}::integer
          WHEN error_status IS NULL THEN NULL
          ELSE ${ResolutionStatus.RESOLVED}::integer
        END
      WHERE error_id = ${recordId}
```